pkg/services/librarypanels/database.go

```diff
@@ -12,6 +12,21 @@ import (
 	"github.com/grafana/grafana/pkg/util"
 )
 
+var (
+	sqlStatmentLibrayPanelDTOWithMeta = `
+SELECT lp.id, lp.org_id, lp.folder_id, lp.uid, lp.name, lp.model, lp.created, lp.created_by, lp.updated, lp.updated_by
+	, 0 AS can_edit
+	, u1.login AS created_by_name
+	, u1.email AS created_by_email
+	, u2.login AS updated_by_name
+	, u2.email AS updated_by_email
+	, (SELECT COUNT(dashboard_id) FROM library_panel_dashboard WHERE librarypanel_id = lp.id) AS connected_dashboards
```

Comment:
Couldn't figure out how to write a better SQL statement here. Couldn't get the Join to work when there are no connections for a Library Panel and dashboard.